packages/@uppy/dashboard/src/components/FileItem/index.js

```diff
@@ -21,14 +16,12 @@ module.exports = function FileItem (props) {
   const isPaused = file.isPaused || false
   const error = file.error || false
 
-  const fileName = getFileNameAndExtension(file.meta.name).name
-  const truncatedFileName = props.isWide ? truncateString(fileName, 30) : fileName
-
   const showRemoveButton = props.individualCancellation
     ? !isUploaded
     : !uploadInProgress && !isUploaded
 
   const dashboardItemClass = classNames(
+    'uppy-u-reset',
```

Comment:
Same here, is it beneficial to add it to the container?

Reply:
It's a <li> tag, so yes, it is in this case.

Reply:
Sorry, thought it was the Dashboard root container.